openapi.yaml

```
@@ -1164,6 +1190,19 @@ components:
           type: string
       required:
         - traq_id
+    PostDMRequest:
+      type: object
+      properties:
+        target-user:
```

多分他のpropertiesはスネークケースになっているので統一してほしいです :pray:
```suggestion
        target_user:
```

---

修正しました！
また間違えてしまった、、、